modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -0,0 +1,78 @@
+= Create a Vector Search Index in Quick Mode
+:page-topic-type: guide
+:description: Use Quick Mode to create a Vector Search index in Couchbase Capella.
+
```

Comment:
Similar comment in the general FTS section--would be helpful to call out what Quick Mode doesn't give me...why/when would I choose Quick Mode over Advanced and vice versa?